packages/block-library/src/page-list-item/editor.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-navigation {
```

Comment:
This file is already used in `page-list`, do we need another copy of it?

Reply:
I think I need to remove some stuff from page list for these ones.